onnxruntime/core/providers/cpu/math/matmul_quant_prepack_base.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/framework/op_kernel.h"
+#include "core/mlas/inc/mlas.h"
+#include "core/providers/common.h"
+#include "core/util/qmath.h"
+
+namespace onnxruntime {
+
+class MatMulQuantPrepackBase : public OpKernel {
```

Comment:
nit: I think the name would be better as MatMulIntegerBase instead of using Quant or adding that Prepack is the only behavior of the base class, but up to you...